File: Is_Prime.py
# Program to check if a number is prime or not

num = int(input())

# To take input from the user
#num = int(input("Enter a number: "))

# prime numbers are greater than 1
if num > 1:
   # check for factors
   for i in range(2,num):
       if (num % i) == 0:
           print("No")
           
           break
   else:
       print("Yes")
       
# if input number is less than
# or equal to 1, it is not prime
else:
   print("No")

File: Is_Prime.c

#include <stdio.h>

int main() { 
   int loop, number;
   int prime = 1;
   
   scanf("%d" ,&number);

   for(loop = 2; loop < number; loop++) {
      if((number % loop) == 0) {
         prime = 0;
      }
   }

   if (prime == 1)
      printf("Yes\n"); 
   else
      printf("No\n");
   return 0;
}